packages/editor/src/client/panel/teaserSelectPanel.tsx

```diff
@@ -178,38 +178,7 @@ export function TeaserSelectPanel({onClose, onSelect}: TeaserSelectPanelProps) {
     })
   }
 
-  function currentFilter() {
-    switch (type) {
-      case TeaserType.Article:
-        return (
-          <Input
-            value={filter.title || ''}
-            onChange={value => setFilter({title: value, published: true})}
-          />
-        )
-      case TeaserType.PeerArticle:
-        return (
-          <Input
-            value={filter.title || ''}
-            onChange={value => setFilter({title: value, published: true})}
-          />
-        )
-      case TeaserType.Page:
-        return (
-          <Input
-            value={filter.title || ''}
-            onChange={value => setFilter({title: value, published: true})}
-          />
-        )
-      case TeaserType.Custom:
-        return (
-          <Input
-            value={filter.title || ''}
-            onChange={value => setFilter({title: value, published: true})}
-          />
-        )
-    }
-  }
+  const updateFilter = (value: string) => setFilter({...filter, title: value})
```

Comment:
This can lead to a bug now technically, should be: `setFilter(oldFilter => {...oldFilter, title: value})` because of batching